.ci/openshift-ci/README.md

```diff
@@ -0,0 +1,116 @@
+* [Overview](#overview)
+* [The configuration files on CI operator](#the-configuration-files-on-ci-operator)
+* [The CI pipeline explained](#the-ci-pipeline-explained)
+    * [Build container images](#build-container-images)
+    * [Create the test cluster](#create-the-test-cluster)
+    * [Install Kata Containers](#install-kata-containers)
+    * [Run tests](#run-tests)
+
+# Overview
+
+The OpenShift CI allows for components to be built and tested in clusters
+provided by the OpenShift community. An entitled component should follow the
+[on-boarding process](https://github.com/openshift/ci-tools/blob/master/ONBOARD.md)
+which involve its configuration in the [CI operator](https://github.com/openshift/ci-operator/blob/master/README.md).
+So this directory contains scripts and files used by the OpenShift CI operator
+to test Kata Containers in that CI environment.
+
+# The configuration files on CI operator
+
+The CI operator behavior is defined in YAML files which are placed at the
+[config directory](https://github.com/openshift/release/tree/master/ci-operator/config)
+in a per-component/repository/branch basis. This directory is organized such as
+the operator can infer the components and their repositories that should be
+tested. And the YAML configuration files should be named in a way that it knows
+the branch as well.
+
+Therefore, there is the `kata-containers` directory under the [CI operator config](https://github.com/openshift/release/tree/master/ci-operator/config)
+to host the configuration files for our project. The configuration files are
+named as `kata-containers/<repository>/kata-containers-<repository>-<branch>.yaml`.
+
+In reality the OpenShift CI pipeline runs as a Kubernetes [Prow](https://github.com/kubernetes/test-infra/tree/master/prow)
+CI/CD job in the [Prow CI instance](https://prow.ci.openshift.org) hosted by
+OpenShift. So, in addition to the CI operator's configuration files, there
+should be created the Prow job configuration files.
+
+The CI operator has this [jobs directory](https://github.com/openshift/release/tree/master/ci-operator/jobs)
+on which Prow job files are placed on also per-component/repository/branch
+basis. Find the `kata-containers` jobs directory under [CI operator jobs](https://github.com/openshift/release/tree/master/ci-operator/jobs/).
+These files can be automatically generated from the CI operator configuration
+files as follows:
+
+```
```

Comment:
Nit: You could specify ` ```sh ` or ` ```bash ` to get syntax highlighting here.

Reply:
done